fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedPointsFormatTest.java

```diff
@@ -95,4 +96,10 @@ public void testOneDimTwoValues() throws Exception {
         TestFDBDirectory.allowAddIndexes();
         super.testOneDimTwoValues();
     }
+
+    @Seeds({@Seed(), @Seed("FA63D2AE2DE7C6B0")})
```

Comment:
I confirmed in the logs that it ran this test with two seeds, and used the same "random" seed both builds:
```
LuceneOptimizedPointsFormatTest > testMultiValued > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedPointsFormatTest.testMultiValued {seed=[C185081D42F0F43C:FA63D2AE2DE7C6B0]} STARTED

LuceneOptimizedPointsFormatTest > testMultiValued > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedPointsFormatTest.testMultiValued {seed=[C185081D42F0F43C:FA63D2AE2DE7C6B0]} PASSED

LuceneOptimizedPointsFormatTest > testMultiValued > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedPointsFormatTest.testMultiValued {seed=[C185081D42F0F43C:15A56C2F8C32B474]} STARTED

LuceneOptimizedPointsFormatTest > testMultiValued > com.apple.foundationdb.record.lucene.codec.LuceneOptimizedPointsFormatTest.testMultiValued {seed=[C185081D42F0F43C:15A56C2F8C32B474]} PASSED
```

Reply:
The `@Seed()` here uses the "random" seed, but in this case, the seed is fixed by the `@Seed` on the class.
Adding `@Seed("FA63D2AE2DE7C6B0")` means that it will always run once with that seed to ensure that we are reproducing this issue. This way, this test will run with two seeds in prb: `C185081D42F0F43C:FA63D2AE2DE7C6B0`  and `C185081D42F0F43C:15A56C2F8C32B474`. The former reproduces the issue this PR is fixing, and the latter (I believe) was the seed originally used, and passing in PRB.